components/script/dom/oscillatornode.rs

```diff
@@ -113,6 +115,19 @@ impl OscillatorNodeMethods for OscillatorNode {
     fn Detune(&self) -> DomRoot<AudioParam> {
         DomRoot::from_ref(&self.detune)
     }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn Type(&self) -> OscillatorType {
+        self.oscillator_type.get()
+    }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn SetType(&self, oscillator: OscillatorType) {
```

Comment:
Because `type` has the `SetterThrows` attribute on the WebIDL, the generated setter returns an `ErrorResult`. That's the reason why you are getting [this](https://travis-ci.com/servo/servo/jobs/196337040#L2529) build error. The function type should be:

```rust
fn SetType(&self, type_: OscillatorType) -> ErrorResult
```

Also, one nit: let's use `type_` instead of `oscillator` for the `OscillatorType` argument, please.